lib/transports/rest2.js

```diff
@@ -250,6 +252,52 @@ class RESTv2 {
     return new ModelClass(data)
   }
 
+  /**
+   * @param {Array[]} data order matters
+   * @return {Arr} merged arr of currencies and names sorted with no pairs repeated
+   * @private
+   */
+
+  _mergeCurrenciesListSymLabel (data) {
+    if (!Array.isArray(data) && data.length !== 3) {
```

Comment:
This should be an or `||`

Reply:
And, because i dont want neither that that is not an array and arrays that are not of length 3

Reply:
true, correcting it.